### PR TITLE
Changes to the tutorial

### PR DESCRIPTION
**Pull request by shielddz**
Added a section for creating a SPA instance. One issue to note here is that the word "Designtime" was removed from the accepted words while it was in the accepted words list before, therefore "Designtime" references were changed in the tutorial aswell.

### CODE REVIEW

File: tutorials/spa-create-automation-api-trigger/spa-create-automation-api-trigger.md [hunk 1]
```diff
@@ -101,24 +101,39 @@ Your **API Key** is now created and ready to be used.
...
-[ACCORDION-BEGIN [Step 3: ](Create service binding)]
-1.  Log in to your **SAP BTP Cockpit** and navigate to your Subaccount.
+[ACCORDION-BEGIN [Step 3: ](Create instance and service binding)]
+1.  Navigate to **SAP BTP Cockpit** subaccount > **Services** > **Instances and subscriptions**.
 
-2.  Create a new **Service Binding** by navigating to **Services** > **Instances and Subscriptions** > **Instances** > Click on your **SAP Process Automation Service** instance > Click **...** > **Create Service Binding**.
+2.  Choose **Create**.
 
-    !![Navigate to Service Bindings creation tab](Step3-1.png)
+    !![navigate to instances and subscriptions](Step3-1.png)
```
Comment (MichaelCzcz) [inline tutorials/spa-create-automation-api-trigger/spa-create-automation-api-trigger.md:109]: Please correct typo.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/spa-create-automation-api-trigger/spa-create-automation-api-trigger.md (hunk 1 above)
Comment (shielddz) [reply tutorials/spa-create-automation-api-trigger/spa-create-automation-api-trigger.md:109]: Hi Michael,

I'm not sure of understanding your comment, do you mean change it to capitals like this: "Navigate to Instances and Subscriptions" ?

Comment (MichaelCzcz) [reply tutorials/spa-create-automation-api-trigger/spa-create-automation-api-trigger.md:109]: Just the N at the beginning.

**Comment by shielddz** [comment]
Typo fixed as requested.

**Pull request merged**